gfx.h

```diff
@@ -1138,38 +1139,26 @@ class GfxInternal
         };
         DXGIFactoryReleaser const factory_releaser(factory);
 
-        DXGI_ADAPTER_DESC1 adapter_desc = {};
         if(adapter != nullptr)
         {
-            IDXGIAdapter1 *adapter1 = nullptr;
-
-            struct DXGIAdapterReleaser
-            {
-                IDXGIAdapter1 *adapter;
-                GFX_NON_COPYABLE(DXGIAdapterReleaser);
-                DXGIAdapterReleaser(IDXGIAdapter1 *adapter) : adapter(adapter) {}
-                ~DXGIAdapterReleaser() { if(adapter) adapter->Release(); }
-            };
-            DXGIAdapterReleaser const adapter_releaser(adapter1);
-
```

Comment:
No longer any need for adapter descriptions as theres now the adapter stored directly internally so this was all removed